src/components/AdminSettings/RecordingServers.vue

```diff
@@ -73,36 +96,55 @@ import debounce from 'debounce'
 
 import Plus from 'vue-material-design-icons/Plus.vue'
 
+import { getCapabilities } from '@nextcloud/capabilities'
 import { showSuccess } from '@nextcloud/dialogs'
 import { formatFileSize } from '@nextcloud/files'
 import { loadState } from '@nextcloud/initial-state'
 
 import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
 import NcNoteCard from '@nextcloud/vue/dist/Components/NcNoteCard.js'
+import NcSelect from '@nextcloud/vue/dist/Components/NcSelect.js'
 import NcTextField from '@nextcloud/vue/dist/Components/NcTextField.js'
 
 import RecordingServer from '../../components/AdminSettings/RecordingServer.vue'
+import { CALL } from '../../constants.js'
 import TransitionWrapper from '../TransitionWrapper.vue'
 
+const recordingConsentCapability = getCapabilities()?.spreed?.features?.includes('recording-consent')
+const recordingConsentOptions = [
+	{ id: CALL.RECORDING_CONSENT.OFF, label: t('spreed', 'Off') },
+	{ id: CALL.RECORDING_CONSENT.REQUIRED, label: t('spreed', 'Enabled') },
+	{ id: CALL.RECORDING_CONSENT.OPTIONAL, label: t('spreed', 'Optional') },
```

Comment:
Hmm it's not "optional" consent, but basically "on conversation level"
Maybe we find  a better wording ;P

Reply:
"Configurable" ?